scripts/cmake/Modules/LibAddMacros.cmake

```diff
@@ -572,6 +573,15 @@ function (generate_manpage NAME)
 			add_dependencies (man man-${NAME})
 		endif (RONN_LOC)
 
+		if (NOT EXISTS "${OUTFILE}")
+			message (WARNING "\nThe file “${MAN_PAGE_LOCATION}” does currently not exist. \
```

Comment:
Please use error, we want to always have all man pages.

Reply:
As far as I can tell, we can not use `ERROR` here, since then generating the man page – via the current build system (e.g. `make`, `ninja`) – will not be possible any more. If we do not add the man page, then the installation will fail with an error anyway, just like it does now.

Reply:
Ahh, ok. As long as the build server fails, I am good.